server.js

```diff
@@ -3,33 +3,62 @@ import bodyParser from 'body-parser'
 import cors from 'cors'
 import mongoose from 'mongoose'
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
+import goldenGlobesData from './data/golden-globes.json'
 // import avocadoSalesData from './data/avocado-sales.json'
 // import booksData from './data/books.json'
 // import netflixData from './data/netflix-titles.json'
 // import topMusicData from './data/top-music.json'
 
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo"
+const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/globes"
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
 mongoose.Promise = Promise
 
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
+// setup mangoose models
+const Category = mongoose.model('Category', {
+  name: String
+})
+
+const Movie = mongoose.model('Movie', {
+  title: String,
+  category: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'Category'
+  }
+})
+
+const seedDatabase = async () => {
+  await Category.deleteMany()
+  await Movie.deleteMany()
+
+  const cat = new Category({ name: 'Best movie of XX century' })
+  await cat.save()
+
+  const mov = new Movie({ title: 'Rejs (English title - The Cruise)', category: cat })
+  await mov.save()
+}
+
+seedDatabase()
```

Comment:
careful though, as this will reset and seed the database every time you start the server